scripts/run-blackbox-tests.sh

```diff
@@ -6,8 +6,8 @@ cert_tool_version=59657b2
 # Run API and DB
 docker-compose pull
 docker-compose --project-name montagu up -d
-docker run docker.montagu.dide.ic.ac.uk:5000/montagu-cert-tool:$cert_tool_version gen-self-signed password > keystore
-./scripts/add-certificate-to-docker.sh keystore montagu_api_1
```

Comment:
This code was left over from when the API used SSL (instead of it being hidden behind a reverse proxy that uses SSL). It was generating a certificate that then wasn't being used.